### PR TITLE
Fix Simultaneous button click

### PR DESCRIPTION
**Pull request by sebavan**
https://forum.babylonjs.com/t/detect-simultaneously-pressed-mousebuttons/6278/7

### CODE REVIEW

**Comment by sebavan** [comment]
Fix #10087

File: src/DeviceInput/deviceInputSystem.ts [hunk 1]
```diff
@@ -355,17 +355,29 @@ export class DeviceInputSystem implements IDisposable {
                 pointer[PointerInput.DeltaVertical] = evt.movementY;
 
                 if (this.onInputChanged) {
+                    // The browser might use a move event in case
+                    // of simultaneous mouse buttons click for instance. So
+                    // in this case we stil need to propagate it.
+                    let fireFakeMove = true;
```
Comment (deltakosh) [inline src/DeviceInput/deviceInputSystem.ts:361]: does it work if you click AND move simulteanously? Don't we swallow one in this case?

Comment (sebavan) [reply src/DeviceInput/deviceInputSystem.ts:361]: not that I have seen so far let me double check

Comment (sebavan) [reply src/DeviceInput/deviceInputSystem.ts:361]: nope all good

**Pull request merged**